graph_shortest_path/routing.py

```diff
@@ -45,16 +50,33 @@ def find_vertex(self, value):
         pass
 
     def bfs(self, start):
+        visited = []
+        vertQueue = Queue()
+        vertQueue.enqueue(start)
+        while vertQueue:
+            currentVert = vertQueue.dequeue()
+        # print(currentVert, end="")
+        for nbr in (self.vertList[currentVert].getConnections()):
+            if nbr not in visited:
+                vertQueue.enqueue(nbr)
+                visited.append(currentVert)
         """
```

Comment:
consider using variable names that have more intuitive meanings...I have no idea what 'nbr' is